docs/components.md

```diff
@@ -4,68 +4,117 @@
-|              [tcplog](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/tcplogreceiver)              |                                                                                                                       |                                                                                                             |                                                                                                                                     |
-| [windowsperfcounters](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/windowsperfcountersreceiver) |                                                                                                                       |                                                                                                             |                                                                                                                                     |
-
-## Experimental/Development
-
-These components should not be considered stable. They are made available
-for testing and validation purposes and may be removed at any time.
-
-| Receivers                                           | Processors                                                                                                                    | Exporters                                     | Extensions |
-|-----------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------|------------|
-| [discovery](../internal/receiver/discoveryreceiver) | [logstransform](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/logstransformprocessor) | [pulsar](../internal/exporter/pulsarexporter) |            |
-|                                                     | [span_metrics](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanmetricsprocessor)    |                                               |            |
-|                                                     | [timestamp](../pkg/processor/timestamp)                                                                                       |                                               |            |
+> Each component has a link to its configuration documentation.
+
+<div style="display: grid;grid-template-columns: auto auto auto auto;">
+
+<div>
```

Comment:
Why are these empty div blocks needed?

Reply:
Just playing - it looks like with these in some markdown envs you can display all the tables side by side. Github doesn't seem to agree though.